utilities/transactions/write_unprepared_txn.cc

```diff
@@ -666,13 +668,21 @@ Status WriteUnpreparedTxn::WriteRollbackKeys(
     return Status::OK();
   };
 
-  for (const auto& cfkey : tracked_keys) {
-    const auto cfid = cfkey.first;
-    const auto& keys = cfkey.second;
-    for (const auto& pair : keys) {
-      auto s = WriteRollbackKey(pair.first, cfid);
-      if (!s.ok()) {
-        return s;
+  if (lock_tracker.IsPointLockSupported()) {
```

Comment:
If both range lock and write unprepared is used together, today it doesn't work right? Is it better to check that the options aren't used together, and assert here that we're using point locks?

Reply:
@lth I changed this to an assertion. @spetrunia When write unprepared supports range lock in the range lock PR, feel free to remove this assertion.

Reply:
@lth, AFAIU:  RangeLocking+WriteUnprepared didn't work together until this commit (by you) https://github.com/facebook/rocksdb/commit/dc23c125c30f9b865ed9663b5d9d6d6ae6548403 fixed this. Now, they should be able to work together, although this doesn't have any test coverage atm.